### PR TITLE
dev-python/jellyfish: add py3.5, doc, tests, EAPI=6

### PR DESCRIPTION
**Pull request by sbraz**
@SoapGentoo Another py3.5 package! I don't think I need a revbump but please tell me if I'm wrong :)

### CODE REVIEW

File: dev-python/jellyfish/jellyfish-0.5.6.ebuild [hunk 1]
```diff
@@ -13,5 +15,29 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
...
+"
+
+python_compile_all() {
+	if use doc; then
+		esetup.py build_sphinx
+		HTML_DOCS=( build/sphinx/html/. )
+	fi
+}
+
+python_compile() {
```
Comment (SoapGentoo) [inline dev-python/jellyfish/jellyfish-0.5.6.ebuild]: `python_compile` before `python_compile_all`

File: dev-python/jellyfish/jellyfish-0.5.6.ebuild [hunk 2]
```diff
@@ -13,5 +15,29 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 LICENSE="BSD-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
```
Comment (SoapGentoo) [inline dev-python/jellyfish/jellyfish-0.5.6.ebuild]: are you missing `USE=doc` stuff (i.e. sphinx) in `DEPEND`?

Comment (sbraz) [inline dev-python/jellyfish/jellyfish-0.5.6.ebuild]: Absolutely :/

File: dev-python/jellyfish/jellyfish-0.5.6.ebuild (hunk 1 above)
Comment (sbraz) [inline dev-python/jellyfish/jellyfish-0.5.6.ebuild]: Why? I mean the latter calls the former, doesn't it?

**Pull request closed without merging**